src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -349,10 +351,12 @@ static void leggTilManuelleFilter(BoolQueryBuilder queryBuilder, Filtervalg filt
         if (filtervalg.harDinSituasjonSvar()) {
             BoolQueryBuilder brukerensSituasjonSubQuery = boolQuery();
             filtervalg.registreringstype.forEach(dinSituasjonSvar -> {
-                if (dinSituasjonSvar == DinSituasjonSvar.INGEN_DATA) {
-                    brukerensSituasjonSubQuery.should(boolQuery().mustNot(existsQuery("brukers_situasjon")));
+                if (dinSituasjonSvar == JobbSituasjonBeskrivelse.INGEN_DATA) {
+                    brukerensSituasjonSubQuery.should(boolQuery().mustNot(existsQuery("brukers_situasjoner")));
                 } else {
-                    brukerensSituasjonSubQuery.should(matchQuery("brukers_situasjon", dinSituasjonSvar));
+                    mapJobbsituasjonTilSituasjonFraGammelRegistrering(dinSituasjonSvar.name()).forEach(sokerOrd ->
```

Comment:
This is bit confusing.. what kind of options we will have in frontend? Are we going to have more options then before for din situasjon or it will be the same?

Reply:
```
export const registreringstype = {
    ER_PERMITTERT: {label: 'Er permittert eller kommer til å bli permittert'},
    USIKKER_JOBBSITUASJON: {label: 'Er usikker på jobbsituasjonen min'},
    HAR_BLITT_SAGT_OPP: {label: 'Jeg har blitt sagt opp av arbeidsgiver'},
    DELTIDSJOBB_VIL_MER: {label: 'Har deltidsjobb, men vil jobbe mer'},
    VIL_BYTTE_JOBB: {label: 'Har jobb, men vil bytte'},
    AKKURAT_FULLFORT_UTDANNING: {label: 'Har akkurat fullført utdanning, militærtjeneste eller annet'},
    HAR_SAGT_OPP: {label: 'Har sagt opp eller kommer til å si opp'},
    ALDRI_HATT_JOBB: {label: 'Har aldri vært i jobb'},
    IKKE_VAERT_I_JOBB_SISTE_2_AAR: {label: 'Har ikke vært i jobb de siste 2 årene'},
    KONKURS: {label: 'Arbeidsgiveren min er konkurs'},
    MIDLERTIDIG_JOBB: {label: 'Jeg har fått midlertidig jobb hos en annen arbeidsgiver'},
    NY_JOBB: {label: 'Jeg skal begynne å jobbe hos en annen arbeidsgiver'},
    ANNET: {label: 'Ingen av situasjonene passer for meg'},
    INGEN_DATA: {label: 'Ingen registreringsinformasjon'}
};
```

De har foreksempel sagt at MISTET_JOBBEN nå blir HAR_BLITT_SAGT_OPP. Vi har endret i frontenden for å ha det likt som de. Men vi har ikke migrert over de gamle registreringene i vår løsning. Derfor må vi sørge for at vi også søker på folk som har de gamle statusene i tillegg slik at ingen faller utenom

Reply:
Maybe it would be nice to improve naming..